src/components/foodDescription/FoodDescription.js

```diff
@@ -13,10 +16,24 @@ function FoodDescription(props) {
     
     const { foodListData } = props;
 
+    const sectionRefs = useRef([])
+
+    function setRef(ele){
+        if(!sectionRefs.current.includes(ele))
+            sectionRefs.current.push(ele);
+    }
+
+    function scrollHandler(category){
+        const section = sectionRefs.current.find((element)=>{
+            return element.innerText === category;
```

Comment:
comparison should either be case insensitive or change it lowerCase then compare